### PR TITLE
Possible Linux Fix

### PR DESCRIPTION
**Pull request by mikamerath**
I've updated the CMake to only use the /EHsc flag when compiling for MSVC, I'm unsure if this is the correct solution so I would like some testing on this for Linux.

### CODE REVIEW

**Comment by mikamerath** [comment]
@katiesweet, can you verify that this fix works?

**Comment by katiesweet** [comment]
@michaelkamerath 
It works! However, there are other problems after this one. Go ahead and pull it in, and I will discuss the other problems with you tonight.

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,9 @@ add_test(NAME UnitTests COMMAND Test)
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}) # Findexternpro.cmake
 set(externpro_REV 16.07.1)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
```
Comment (katiesweet) [inline CMakeLists.txt]: Just kidding, you can't have this line in here without the if statement around it for Linux.

**Review by katiesweet — CHANGES_REQUESTED** [review]

**Comment by mikamerath** [comment]
@katiesweet, please verify that this works correctly. @Shemarama @davidbrowning for reference, an example of how to serialize and deserialize cards can be found in the test.cpp

**Comment by katiesweet** [comment]
@michaelkamerath, it works for me! Thanks for doing this fix.

**Review by Frangello — CHANGES_REQUESTED** [review]
@michaelkamerath On the Card.hpp file, delete `Card::` before `serialize(Archive& ar, const unsigned int version)`
It is not valid code, only Visual Studios will compile.

**Comment by mikamerath** [comment]
@Frangello, thanks for catching that. It has been done

**Pull request merged**